### ISSUE TITLE
Instructions for MDX format

### ISSUE THREAD

**Issue by binhxn** [pocka/storybook-addon-designs#22]
Is there a way to implement this addon in MDX format?

Here's an example with `@storybook/addon-knobs` :

```
<Story name="knobs" parameters={{ decorators: [withKnobs] }}>
  <Avatar
    loading={boolean('Loading')}
    size={select('Size', ['tiny', 'small', 'medium', 'large'])}
    username="Some user"
    src="https://i.pravatar.cc/300"
  />
</Story>
```

Resources: 
- You can see the code snippet above [here](https://www.learnstorybook.com/design-systems-for-developers/react/en/document/)
- Storybook does not have official MDX docs as of date, but they have provided a link to the [technical review](https://docs.google.com/document/d/1un6YX7xDKEKl5-MVb-egnOYN8dynb5Hf7mq0hipk8JE/edit) of it.

edit: add resources

**Comment by pocka** [comment]
@binhxn 
All you need is putting the decorator (`withDesign`) and setting the `design` parameter. According to the [docs in `next` branch](https://github.com/storybookjs/storybook/blob/next/addons/docs/docs/mdx.md#decorators-and-parameters), you can use this addon by:

```jsx
// `decorators={[withDesign]}` is not needed when you put it as a global decorator in the config file
<Story
  name="story"
  decorators={[withDesign]}
  parameters={{
    design: {
      type: 'figma',
      url: 'https://your-figma-file'
    }
  }}
>
  <YourComponent />
</Story>
```

NOTE: I don't test this 😎  If it doesn't work please tell me.

**Comment by shilman** [comment]
LGTM @pocka 👍

**Comment by binhxn** [comment]
@pocka @shilman Thank you for the speedy response.
However, when I add the decorator to either the global, component, or story level, I run into a build error.

<img width="807" alt="Screen Shot 2019-10-19 at 8 40 05 AM" src="https://user-images.githubusercontent.com/8155387/67147750-85ef4e80-f24c-11e9-8e12-a13271534af3.png">

1. When I add as global decorator in my `config.js` file:
```
```

**Comment by shilman** [comment]
Here's an example that's working for me at the component level:

```jsx
import { Meta, Story } from "@storybook/addon-docs/blocks";
import { config, withDesign } from "storybook-addon-designs";
import { Button } from "./Button";

<Meta title="Test" decorators={[withDesign]} />

<Story
  name="button"
  parameters={{
    design: config({
      type: "iframe",
      url: "https://www.wikipedia.org"
    })
  }}
>
  <Button>hello</Button>
</Story>
```

**Comment by binhxn** [comment]
@shilman Thank you, it worked! Well, with some alterations:

1. ~~I originally had the `<Preview>` component (from **addon-docs**) wrapped around `<Story>`. It did not work, so I removed it.~~ Somehow worked while I was typing this out
2. Adding the `config` function fixed it for me. I'm now able to see the preview.
3. Adding the `withDesign` decorator to `<Meta>` broke it for me completely. I have to remove it in order to work. It shows the same error as the screenshot I posted above.

So using your code above, this is what worked for me:

```
import { Meta, Story } from "@storybook/addon-docs/blocks";
import { config } from "storybook-addon-designs";
import { Button } from "./Button";

<Meta title="Test" />

<Story
  name="button"
  parameters={{
    design: config({
      type: "iframe",
      url: "https://www.wikipedia.org"
    })
  }}
>
  <Button>hello</Button>
</Story>
```

Regardless, it solved my problem so I will close this ticket. Thank you again!
